api/docker/php/docker-entrypoint.sh

```diff
@@ -11,56 +11,36 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ] || [ "$1
-			echo "Generating public / private keys for JWT"
-			mkdir -p config/jwt
-			echo "$jwt_passphrase" | openssl genpkey -out config/jwt/private.pem -pass stdin -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096
-			echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -out config/jwt/public.pem -pubout
-			setfacl -R -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
-			setfacl -dR -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
-		fi
+	if [ "$APP_ENV" = 'prod' ]; then
+	  ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
+	  setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
+	  setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var
 	fi
 
 	if [ "$APP_ENV" != 'prod' ]; then
-		composer install --prefer-dist --no-progress --no-interaction
-	fi
-
-	if grep -q DATABASE_URL= .env; then
-		echo "Waiting for database to be ready..."
-		ATTEMPTS_LEFT_TO_REACH_DATABASE=60
-		until [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ] || DATABASE_ERROR=$(php bin/console dbal:run-sql -q "SELECT 1" 2>&1); do
-			if [ $? -eq 255 ]; then
-				# If the Doctrine command exits with 255, an unrecoverable error occurred
-				ATTEMPTS_LEFT_TO_REACH_DATABASE=0
-				break
-			fi
-			sleep 1
-			ATTEMPTS_LEFT_TO_REACH_DATABASE=$((ATTEMPTS_LEFT_TO_REACH_DATABASE - 1))
-			echo "Still waiting for database to be ready... Or maybe the database is not reachable. $ATTEMPTS_LEFT_TO_REACH_DATABASE attempts left."
-		done
+    if [ ! -f config/jwt/private.pem ]; then
+      jwt_passphrase=${JWT_PASSPHRASE:-$(grep ''^JWT_PASSPHRASE='' .env | cut -f 2 -d ''='')}
+      if ! echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -noout > /dev/null 2>&1; then
+        echo "Generating public / private keys for JWT"
+        mkdir -p config/jwt
+        echo "$jwt_passphrase" | openssl genpkey -out config/jwt/private.pem -pass stdin -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096
+        echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -out config/jwt/public.pem -pubout
+        setfacl -R -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
+        setfacl -dR -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
+      fi
+    fi
 
-		if [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ]; then
-			echo "The database is not up or not reachable:"
-			echo "$DATABASE_ERROR"
-			exit 1
-		else
-			echo "The database is now ready and reachable"
-		fi
+		composer install --prefer-dist --no-progress --no-interaction
 
-		if ls -A migrations/*.php >/dev/null 2>&1; then
-			php bin/console doctrine:migrations:migrate --no-interaction
-		fi
+    if grep -q DATABASE_URL= .env; then
+      migrate-database || exit 1
 
-		if ls -A fixtures/*.yml >/dev/null 2>&1; then
-			php bin/console hautelook:fixtures:load --quiet
+			if ls -A fixtures/*.yml >/dev/null 2>&1; then
+				php bin/console hautelook:fixtures:load --quiet
```

Comment:
How do we ensure, this doesn't remove existing data? Are fixtures only loaded, if database is empty?

Reply:
Fixtures aren't loaded in the deployments for now. For production seed data, [database migrations are the recommended way on the internet](https://stackoverflow.com/questions/47902191/how-do-use-database-fixtures-for-production-environment/47902192#47902192).

Reply:
Didn't recognize the if above. So eventually we could be using:
- fixtures: Unit tests (and local development)
- migrations: Seed data (for production)
- migrations: additional seed/test data for local development & `dev` deployment

Reply:
I tried to summarize our options in #2371